package.json

```diff
@@ -83,7 +87,9 @@
     "react-test-renderer": "~17.0.1",
     "recursive-copy": "^2.0.11",
     "scheduler": "^0.20.1",
+    "ts-jest": "^26.4.3",
```

Comment:
Could you explain why you added the `ts-jest` dependency here?

Reply:
It was added because we thought this was needed as part of jest dependency in running tests written with ts. But, I tried running without it and it works fine. I've removed it. Thanks 👍